docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -2,44 +2,29 @@
+
+//[NOTE]
+//The following example scenario describes a single-cell setup. Real
+//multi-stack topology that is recommended for production use results in cells having a different database layout, and should use different naming schemes. kgilliga: We might reinstate this note after multi-cell is finished in Feature Release 1.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- ** the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB;
- ** the xref:adopting-the-identity-service_adopt-control-plane[Adopting the Identity service] needs to be imported;
- ** the xref:adopting-the-key-manager-service_adopt-control-plane[Adopting the Key Manager service] needs to be imported;
- ** the xref:adopting-the-placement-service_{context}[Adopting the Placement service] needs to be imported;
- ** the xref:adopting-the-image-service_adopt-control-plane[Adopting the Image service] needs to be imported;
- ** the xref:migrating-ovn-data_migrating-databases[Migrating OVN data] need to be imported;
- ** the xref:adopting-the-networking-service_adopt-control-plane[Adopting the Networking service] needs to be imported;
-** the {bare_metal} needs to be imported;
-//kgilliga:I removed the link because it did not work. I might rewrite this section anyway.
- ** Required topology-specific service configuration. For more information, see
-xref:proc_retrieving-topology-specific-service-configuration_migrating-databases[Retrieving topology-specific service configuration].
- ** {rhos_prev_long} services have been stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
+* Define the following shell variables. Replace the following example values with the values that are correct for your environment:
 ----
-alias openstack="oc exec -t openstackclient -- openstack"
+$ alias openstack="oc exec -t openstackclient -- openstack"
 ----
 
 .Procedure
 
+. Patch the `OpenStackControlPlane` CR to deploy the {compute_service}:
++
 [NOTE]
-This procedure assumes that {compute_service} Metadata is deployed on the top level and not on each cell level, so this example imports it the same way. If the source deployment has a per cell metadata deployment, adjust the given below patch as needed. Metadata service cannot be run in `cell0`.
-
-
-. Patch `OpenStackControlPlane` to deploy the {compute_service}:
+This procedure assumes that {compute_service} metadata is deployed on the top level and not on each cell level. If the {OpenStackShort} deployment has a per-cell metadata deployment, adjust the following patch as needed. Metadata service cannot be run in `cell0`.
```

Comment:
Suggestion: use active voice:

You cannot run the metadata service in `cell0`.